data/json/items/resources/tailoring.json

```
@@ -246,7 +255,8 @@
     "name": { "str": "rigid kevlar plate" },
     "description": "This is a compressed panel of Kevlar treated with epoxy or other adhesive.  It could be used to repair items made of Kevlar.",
     "weight": "300 g",
-    "volume": "250 ml",
+    "volume": "235 ml",
+	"//": "Density of 1.4 grams per mL + 10% for being rigid and possibly oddly-shaped",
```

```suggestion
	  "//": "Density of 1.4 grams per mL + 10% for being rigid and possibly oddly-shaped",
```
Note that the epoxy may be increasing the density.